WISE-Community/WISE-Client
# feat(User): Change role to roles

## Changes
- User object now has a roles string array field instead of a role string field
- Updated all the places that look at the role field and make them work with the roles array instead

## Test
- Test with https://github.com/WISE-Community/WISE-API/pull/258
- Make sure the header displays the correct links depending on the roles
- Make sure the header account menu displays the correct elements depending on the roles

Closes #1656
